flake.nix

```
@@ -1,21 +1,25 @@
 {
   description = "nixos-generators - one config, multiple formats";
 
+  # Lib dependency
+  inputs. = "github:divnix/nixpkgs.lib";
```

Don't you miss that?
```suggestion
  inputs.nixlib.url = "github:divnix/nixpkgs.lib";
```

---

ooops :smile: -- four eyes better than two. Thanks!